packages/utils/tokens/src/shadow-tokens.ts

```diff
@@ -0,0 +1,23 @@
+import { ColorValue } from 'react-native';
+import { IOperationSet } from '@uifabricshared/foundation-tokens';
```

Comment:
Just an FYI that uifabricshared packages are generally deprecated - we're trying to use other packages instead now. For the Theme type you can probably take it from @fluentui-react-native/framework instead